### PR TITLE
DOC-669: Replace pwa with graphql

### PR DESCRIPTION
**Pull request by jeff-matthews**
The purpose of this pull request (PR) is to change the metadata that we use to designate a feature as "PWA Studio only" to "GraphQL metapackage only".

According to the product owner (Cris Partica), "pwa" is too specific. Features like [contactUs](https://devdocs.magento.com/guides/v2.4/graphql/mutations/contact-us.html) currently marked with "pwa only" are actually available for any headless storefront solution, including (but not limited to) AEM, Hyvä, and Vue Storefront.

This is part of a larger doc task described in DOC-669 (related to #98). 

I've been trying to see if the results of this PR work as expected locally, but I'm unable to do so. These are the steps that I'm following:

1. Create a new branch in the devdocs-theme repo.
2. Modify theme.
3. Update the theme version manually in multiple files.
4. Recompile CSS and JS using the npm scripts:
   - `npm run build:css`
   - `npm run build:js`
 5. Modify the Gemfile in my [local devdocs branch](https://github.com/magento-devdocs/devdocs/tree/DOC-669-graphql-metapackage) as described [here](https://github.com/magento-devdocs/devdocs-theme#how-to-apply-the-theme-to-your-project) (using latest commit instead of branch).
 6. Install theme in devdocs branch (bundle install) and build locally with new graphql-only metadata set.

**Expected results:**

New topic with metadata renders correctly: http://127.0.0.1:4000/extensions/graphql/

**Actual results:**

Topic renders with no metadata label.

### CODE REVIEW

**Comment by jeff-matthews** [comment]
The staging preview adds the blue dot to the TOC, but doesn't add the label to the top of the topic: https://devdocs.magedevteam.com/614/extensions/graphql/

File: _includes/layout/edition-callout.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
 {% endif %}
 {% if page.pwa_only == true %}
```
Comment (belbiy) [inline _includes/layout/edition-callout.html:25]: Probably needs to be changed to `page.graphql_only`? This page variable that is set on the whole page. Edition callout on top of the page uses that

**Comment by jeff-matthews** [comment]
Aha! Thank you @belbiy!!!!

**Comment by jeff-matthews** [comment]
Local builds now work as expected. I'm running a staging build right now and will add a link when it's ready.

**Comment by jeff-matthews** [comment]
https://devdocs.magedevteam.com/615/extensions/graphql/

**Comment by dshevtsov** [comment]
This change will be applied across User Guides as well. I'm not sure if `GraphQL metapackage only` is clear for User guide readers. Also, the `Learn more` opens [Product editions](https://docs.magento.com/user-guide/getting-started.html#product-editions) which have to explain what is _GraphQL metapackage_ (edition?).

**Comment by jeff-matthews** [comment]
That's true. I have a branch in the merchdocs repo that updates that page, but I forgot to create a draft PR.

I think that page is the key for explaining what the metapackage is. I don't see an alternative.

**Comment by jeff-matthews** [comment]
Closing associated Jira issue as "Canceled".

**Pull request closed without merging**